pootle/apps/pootle_fs/resources.py

```diff
@@ -163,7 +163,7 @@ def pootle_changed(self):
 
     def reload(self):
         """Uncache cached_properties"""
-        for k, v in self.__dict__.items():
+        for k, __ in self.__dict__.items():
```

Comment:
I've only used `$var_` if I thought it added value to understanding.  I think it's pretty clear here that this is `key`, `value`. And I don't think it helps here.